### PR TITLE
Test 1 - Add sort test in Products Section

### PR DESCRIPTION
**Pull request by DanyloHanenko**

### CODE REVIEW

File: src/tests/inventory-sorting.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    // 3. Sorting by price (low to high)
+    // 4. Sorting by price (high to low)
+
+    Object.entries(SORT_OPTIONS).forEach(([sortOption, sortFunction]) => {
+        test(`should sort products by ${sortOption}`, async (
+            /** @type {{ app: import('../pages/Application').Application }} */{ app },
+        ) => {
+            await app.inventory.select.selectOption({ value: sortOption });
+            const currentProducts = await app.inventory.getProductData();
+            const sortedProducts = [...initialProducts].sort(sortFunction);
```
Comment (EugenePetrik) [inline src/tests/inventory-sorting.spec.js:44]: `[...initialProducts]` we do not use spread operator here, this should work

```
const sortedProducts = initialProducts.sort(sortFunction);
```

**Review by EugenePetrik — APPROVED** [review]
Good job! Approved, but left a small comment

File: src/tests/inventory-sorting.spec.js (hunk 1 above)
Comment (DanyloHanenko) [reply src/tests/inventory-sorting.spec.js:44]: This is a common practice to eliminate side-effects inside tests.

I added spread operator to eliminate changing the data by the test execution.
If the each test changes a global variable, changing the order of tests can affect it and we will have unwanted mutation of data. In the currect test, it won't be a case, but for a sake of future modification or updating the test, it's the best to have spread operator here.

So I'm cosidering stay it as it is.

Comment (EugenePetrik) [reply src/tests/inventory-sorting.spec.js:44]: Ok, sounds good.

**Comment by ovartem** [comment]
closing this PR as it was created into mine repo

**Pull request closed without merging**